ginga/examples/gw/plot_time_series.py

```diff
@@ -0,0 +1,364 @@
+#! /usr/bin/env python3
+#
+"""
+Plot time series data in an interactive plot viewer.
+
+Usage
+=====
+$ python3 plot_time_series.py --loglevel=20 --stderr
+
+Plots add new data every second, and update on screen every 5 sec.
+24 hours of data is kept.  Each plot starts out in "autoaxis X PAN"
+and "autoaxis Y PLOT".
+
+Things you can do in a plot:
+
+1) Scroll the mouse wheel to zoom the X axis
+
+2) Hold CTRL and scroll the mouse wheel to zoom the Y axis.
+
+3) Press 'f' to autozoom the Y axis to the full range of the Y data.
```

Comment:
Is this a typo? f key didn't do anything for me.
```suggestion
3) Press 'v' to autozoom the Y axis to the full range of the Y data.
```

Reply:
Not in sync with the code, should be 'y' and 'v', not 'f' and 'y'.  Will fix.